### ISSUE TITLE
Dragging minimap ignores invert_viewport_drag option

### ISSUE THREAD

**Issue by qqkookie** [OpenRCT2/OpenRCT2#7900]
**OS:** Windows 10
**Version:** 0.2.0
**Commit/Build:**

Dragging minimap with right mouse pressed ignores the `invert_viewport_drag option`. Regardless of the invert_viewport_dragoption setting, it always moves opposite direction of dragging, the original vanilla RCT behavior. It should honor the `invert_viewport_drag option` option.

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
1. 
2. 

**Dump file**

**Screenshots / Video:**

**Save game:**

**Comment by Broxzier** [comment]
The map window does not use a viewport, but a scrollable area. Just like other scrollable areas, right-mouse dragging on them moves the scrollbars around. I don't think switching this around depending on the viewport drag option should be added.

**Comment by qqkookie** [comment]
Dragging scrollbar handle of minimap window with **left** mouse button pressed is different from dragging the minimap itself with **right** mouse button pressed.  I modified it so that  minimap  obey the invert_viewport_drag option. Then, dragging the minimap itself  with right mouse pressed  and moving in upward direction will  move the minimap itself upward. But the scrollbar handle moves downward, as expected.  It is very natural way of movement.

**Comment by Broxzier** [comment]
That change will affect _all_ the scroll areas, like the scenery window and guest list, not just the map window. This behaviour is to give a sense of swiping through the lists like how you would do with a touchscreen.

**Comment by qqkookie** [comment]
Yes, it is exactly what I would like. Now all scrollable lists feel like that I grab the list with the mouse right button and dragging it and move around.  The original inverted movement always is felt awkward to me.  Please, try yourself. You will love it.
